src/analysis/RDFatomistic.cpp

```diff
@@ -40,41 +40,34 @@ using namespace std;
 
 namespace espressopp {
   namespace analysis {
-    
-    // TODO currently works correctly for Lx = Ly = Lz
+
+    // TODO currently works correctly if L_y is shortest side length (it's fine if L_x or L_z are equal)
```

Comment:
You mean if L_x or L_z are equal to L_y, right? Not if they're equal to each other? Also, we should explictly mention this limitation in the documentation in .py